config/jobs/kubernetes-sigs/kjob/OWNERS

```diff
@@ -0,0 +1,9 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+- mimowo
+- mwielgus
+reviewers:
+- mimowo
+- mwielgus
+- mbobrovskyi
```

Comment:
To unblock this PR, you may need to @mbobrovskyi from reviewers. I still suggest you open up a request to join the kubernetes-sig org. Happy to sponsor you but we may want this in.

Reply:
Ah, I see. I've removed it for now.